### DISCUSSION TITLE
How to add my hysteria config in clash.meta ?

### DISCUSSION THREAD

**Discussion by MintouZintou** [apernet/hysteria#1065] [Q&A]
I am trying to find a good client for hy2 on Android. 
The best I could find was meta (nekobox don't work on my model). But could you help me ?
I don't know how to add by config to clash, it asks for provider files provider files + config in yaml extension, which I clearly don't have. I only have a server with hysteria2 on it.

**Comment by TommcyOWO** [comment]
try v2rayNG ?

**Reply by MintouZintou** [reply]
V2rayng doesn't support hy2 protocol (hysteria2)

**Comment by chidedao** [comment]
try like this
- {"name":"hy2test","type":"hysteria2","server":"yourserverl.com","ports":"20000-50000","password":"yourpassword"}
